server/common/admin-rest-client/src/main/java/org/kaaproject/kaa/server/common/admin/AdminClient.java

```diff
@@ -1062,26 +1062,25 @@ public UserDto editUserProfile(UserDto userDto) {
         return response.getBody();
     }
 
-    public CredentialsDto provisionCredentials (String applicationId, String credentials) {
+    public CredentialsDto provideCredentials (String applicationId, String credentialsBody) {
```

Comment:
I'd concur that "provision" is a more appropriate, widely used term for adding trusted security credentials and the appropriate metadata. "Provide" is more abstract and will require more explanations.